RiscVPkg/Library/RiscVDxeIplHandoffOpenSbiLib/RiscVDxeIplHandoffOpenSbiLib.c

```diff
@@ -18,22 +18,22 @@
 #include <Library/RiscVPlatformDxeIpl.h>
 
 /**
-   RISC-V platform DXE IPL to DXE OpenSBI mdoe switch handler.
+   RISC-V platform DXE IPL to DXE OpenSBI mode switch handler.
    This function is executed in RISC-V Supervisor mode.
+   It is called by sbi_init()
 
    This function performs a CPU architecture specific operations to execute
    the entry point of DxeCore with the parameters of HobList.
+   // Question: Doesn't seem to install it, does it?

```

Comment:
cc @changab 
Is that missing?

Reply:
Ah yes.
we missed installing END_OF_PEI.
PeiServicesInstallPpi (&gEndOfPeiSignalPpi);

Reply:
Hm, when I add it, it gets stuck there. I'll investigate that. WIP here: #4